doc/user-guides/secure-protect-connect-single-multi-cluster.md

```diff
@@ -0,0 +1,703 @@
+```
+
+### Pre Req
+
+- Install `kuadrantctl`. You can find a compatible binary and download it from the [kuadrantctl releases page](https://github.com/Kuadrant/kuadrantctl/releases/tag/v0.2.2)
+- Ability to distribute resources generated via `kuadrantctl` to multiple clusters, as though you are a platform engineer.
+
+### Setup HTTPRoute and backend
+
+Copy at least one of the following example OAS to a local location:
+
+[sample OAS rate-limiting and API Key spec](../../examples/oas-apikey.yaml)
+
+[sample OAS rate-limiting and OIDC spec](../../examples/oas-oidc.yaml)
+
+Setup some new env vars:
+
+```bash
+export oasPath=examples/oas-apikey.yaml
+# Ensure you still have these environment variables setup from the start of this guide:
+export rootDomain=example.com
+export gatewayNS=api-gateway
+```
+
+### Use OAS to define our HTTPRoute rules
+
+We can generate Kuadrant and Gateway API resources directly from OAS documents, via an `x-kuadrant` extension.
+
+> **Note:** for a more in-depth look at the OAS extension take a look at our [kuadrantctl documentation](https://docs.kuadrant.io/kuadrantctl/).
+
+Let's use `kuadrantctl` to generate our `HTTPRoute`.
+
+> **Note:** the sample OAS has some placeholders for namespaces and domains - we will inject valid values into these placeholders based on our previous env vars
+
+Generate the resource from our OAS, (`envsubst` will replace the placeholders):
+
+```bash
+cat $oasPath | envsubst | kuadrantctl generate gatewayapi httproute --oas -
+```
+
+If we're happy with the generated resource, let's apply it to the cluster:
+
+```bash
+cat $oasPath | envsubst | kuadrantctl generate gatewayapi httproute --oas - | kubectl apply -f -
+```
+
+Check our new route:
+
+```bash
+kubectl get httproute toystore -n ${$devNS} -o=yaml
+```
+
+We should see that this route is affected by the `AuthPolicy` and `RateLimitPolicy` defined as defaults on the gateway in the gateway namespace.
+
+```yaml
+- lastTransitionTime: "2024-04-26T13:37:43Z"
+        message: Object affected by AuthPolicy demo/external
+        observedGeneration: 2
+        reason: Accepted
+        status: "True"
+        type: kuadrant.io/AuthPolicyAffected
+- lastTransitionTime: "2024-04-26T14:07:28Z"
+        message: Object affected by RateLimitPolicy demo/external
+        observedGeneration: 1
+        reason: Accepted
+        status: "True"
+        type: kuadrant.io/RateLimitPolicyAffected        
+```
+
+### Test connectivity and deny-all auth 
+
+We'll use `curl` to hit an endpoint in the toystore app. As we are using LetsEncrypt staging in this example, we pass the `-k` flag:
+
+```bash
+curl -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${$devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
+```
+
+We are getting a `403` because of the existing default, deny-all `AuthPolicy` applied at the Gateway. Let's override this for our `HTTPRoute`.
+
+Choose one of the following options:
+
```

Comment:
```suggestion
### Test connectivity and deny-all auth 

You will use `curl` to hit an endpoint in the toystore app. Because this example uses Let's Encrypt staging, you can pass the `-k` flag:

```bash
curl -s -k -o /dev/null -w "%{http_code}" "https://$(kubectl get httproute toystore -n ${$devNS} -o=jsonpath='{.spec.hostnames[0]}')/v1/toys"
```

You are getting a `403` because of the existing default, deny-all `AuthPolicy` applied at the Gateway. You can override this for your `HTTPRoute`.

Choose one of the following options:

```